src/lime.h

```diff
@@ -76,6 +76,7 @@
 #define N_RAN_PER_SEGMENT       3
 #define FAST_EXP_MAX_TAYLOR	3
 #define FAST_EXP_NUM_BITS	8
+#define N_SMOOTH_ITERS		20
```

Comment:
@allegroLeiden Should we document this in the user manual? Or is it considered as an internal parameter?